content/contribute/editorial-guide.md

```diff
@@ -0,0 +1,77 @@
+---
+This document describes the responsibilities for the person or people assigned to the editor role for an open source story, as outlined in the editing section of the `new-story` issue template.
+
+> Would you like to be an editor for Open Source Stories? Let us know by sending us an email at [contribute@opensourcestories.org](mailto:contribute@opensourcestories.org?subject=I%27d%20like%20to%20volunteer%20as%20an%20editor!)!
+
+## Editing process
+
+As the editor for a new story, you will be given access to a folder containing the following resources:
+
+* a transcript in the desired output template
+* the audio recording
+* subtitle files
+
+All of your tasks will be completed in the transcript document and tracked in the GitHub issue.
+
+Editing a story can be broken down into the following steps:
+
+* edit transcript for readability
+* break into paragraphs
+* fine-tune title
+* add logical headings
+* add a summary
+* add relevant links (optional)
+* suggest tags (see [opensourcestories.org/tags](https://www.opensourcestories.org/tags) for existing tags)
+* write social media copy (optional)
+
+### Edit transcript for readability
+
+The transcript is what will be posted on [opensourcestories.org](https://www.opensourcestories.org/), but can be edited for clarity. This means that filler words (ums, ahs, like, etc…) may be removed if they impact the readability of the story. A suggested process is:
+
+1. read through the transcript and correct misspellings, grammatical errors, or mistranscriptions
+1. add punctuation where needed
+1. tighten up language and clarify ambiguities
+
+### Break into paragraphs
+
+While the transcript will already have some paragraphs based on speaker and timing, if there are additional breaks that you find would be helpful, please add them.
+
+### Fine-tune title
+
+The transcript will have a draft title, but please feel free to fine-tune it if you have a more appropriate one.
+
+### Add logical headings
+
+Even short stories can use some structure! Please add 3-5 headings (H2) in the story as the conversation progresses. You can look at [prior stories](https://www.opensourcestories.org/stories/) to get a sense for how we have broken them up in the past.
+
+### Add a summary / description
+
+The summary does not appear on the story page, but _does_ appear in the story listing, and it populates the summary for the corresponding podcast episode. Add a 1-3 sentence summary of the conversation, highlighting salient points.
+
+### Add relevant links (optional)
+
+If there are references to people, technologies, or events that you think could benefit from a reference, please add them as links in the transcript.
+
+### Suggest tags (optional)
+
+Tags help people find other interesting stories! Add the tags you think would be most helpful for others.
+
+See [opensourcestories.org/tags](https://www.opensourcestories.org/tags/) for existing tags
+
+### Write social media copy (optional)
```

Comment:
Bookmark for when we create a "social-guidelines" doc -

Add in section on images, copyright, best practices for alt-text.

Reply:
Converted to issue!